ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/hosts.ini

```diff
@@ -95,6 +95,7 @@ sflow
 snmp
 snmp-server-traps
 spanning-tree
+spanning-tree-2
```

Comment:
Please rename `spanning-tree-2` into `spanning-tree-<something-meaningful>`.
I'd also consider renaming original `spanning-tree` into `spanning-tree-<something-meaningful>`. But that's optional, as we have `snmp` and `snmp-<something>` just above.

Reply:
spanning-tree-2 changed to spanning-tree-bpduguard. Or spanning-tree-bpdu sound good?

Reply:
Both options a good for me.